tests/test_hf_api.py

```diff
@@ -4074,6 +4074,10 @@ def test_user_overview(self) -> None:
         assert overview.num_upvotes > 10
         assert len(overview.orgs) > 0
         assert any(org.name == "huggingface" for org in overview.orgs)
+        assert isinstance(overview.num_following, (int, type(None)))
+        assert overview.num_following is None or overview.num_following >= 0
+        assert isinstance(overview.num_followers, (int, type(None)))
+        assert overview.num_followers is None or overview.num_followers >= 0
```

Comment:
```suggestion
        assert overview.num_following > 300
        assert overview.num_followers > 1000
```

We want to have a meaningful test. In our case, `julien-c` has many following and followers so it's preferable to test that. I realized the API is not yet released (should be the case in 1-2 days), that's why you are currently getting `None`. Let's wait a bit and we'll merge once the API is proded.